beacon_node/http_api/src/lib.rs

```diff
@@ -1405,21 +1420,30 @@ pub fn serve<T: BeaconChainTypes>(
         .and(warp::path("beacon"))
         .and(warp::path("blocks"))
         .and(warp::path::end())
-        .and(warp_utils::json::json())
+        .and(warp::body::json())
+        .and(consensus_version_header_filter)
```

Comment:
I think this is technically a breaking change, but this endpoint is deprecated anyway and we should probably delete it (and update our tests not to use it).

Reply:
I've opened an issue to just delete these endpoints:

- https://github.com/sigp/lighthouse/issues/7478